docs/README

```diff
@@ -0,0 +1,35 @@
+# How to build Sphinx documentation
+
+In order to build documentation for the standard library, first install sphinx
+
+```bash
+python -m venv build/
+source build/bin/activate
+pip install sphinx
+
+```
+
+Next, build gem5, then cd to the docs directory
+
+```bash
+scons build/ALL/gem5.opt
+cd docs
+```
+
+If you don't already have all of the rst files in the directory, or need to generate new files, run `../build/ALL/gem5.opt gem5-sphinx-apidoc -o . ../src/python/gem5 -F -e`
```

Comment:
If these are auto generated, then should we remove them from the repository?

We should only check in files that can't be generated.